riscv/tests/riscv.rs

```diff
@@ -65,12 +65,11 @@ fn test_keccak() {
 }
 
 #[test]
-#[ignore = "Too slow"]
```

Comment:
This makes it run in debug mode - I don't think we should do that, should we?

Reply:
ah right, didn't notice that release was already running with `--ignored`